src/Migrations/M022UpdatePostRevisions.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace P4\MasterTheme\Migrations;
+
+use P4\MasterTheme\MigrationRecord;
+use P4\MasterTheme\MigrationScript;
+
+/**
+ * Update the limit of post revisions.
+ */
+class M022UpdatePostRevisions extends MigrationScript
+{
+    /**
+     * Perform the actual migration.
+     *
+     * @param MigrationRecord $record Information on the execution, can be used to add logs.
+     * phpcs:disable SlevomatCodingStandard.Functions.UnusedParameter -- interface implementation
+     */
+    protected static function execute(MigrationRecord $record): void
+    {
+        global $wpdb;
+
+        // Check if exist
+        $MAX_REVISIONS = get_option('revisions_to_keep');
+        if (false === get_option('revisions_to_keep')) {
+            // If not, set 20 by default
+            $MAX_REVISIONS = 20;
+            update_option('revisions_to_keep', $MAX_REVISIONS);
+        }
+
+        $sql = '
+            DELETE FROM wp_posts
```

Comment:
This query removes all the revisions that exceeds the maximum for each post.

For example,
if a post has 22 revisions and another one has 54 revisions, this query will remove 36 unnecessary revisions. 2 for the first post, and 34 for the other post.